src/RescriptVersions.res

```
@@ -37,10 +19,12 @@ let promptVersions = async () => {
 
   s->P.Spinner.start("Loading available versions...")
 
-  let (rescriptVersions, rescriptCoreVersions) = await Promise.all2((
-    getPackageVersions("rescript", rescriptVersionRange),
-    getPackageVersions("@rescript/core", rescriptCoreVersionRange),
-  ))
+  let (rescriptVersions, rescriptCoreVersions) = try await Promise.all2((
+    NpmRegistry.getPackageVersions("rescript", rescriptVersionRange),
+    NpmRegistry.getPackageVersions("@rescript/core", rescriptCoreVersionRange),
+  )) catch {
+  | _exn => Error.make("Fetching versions from registry failed.")->Error.raise
```

need to raise the error here (and not inside getPackageVersions) so that the message shows up nicely formatted in the prompt